ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -853,31 +853,126 @@ $defs:
             type: str
             format: mac
             #required: true # This cannot be required in defaults, since it could be set on any level.
-          inband_management_subnet:
+          inband_mgmt_interface:
             documentation_options:
               filename: Fabric Topology
-              table: Inband management VLAN
+              table: Inband management
             description: |
-              Optional IP subnet assigned to Inband Management SVI on l2leafs in default VRF.
-              Parent l3leafs will have SVI with "ip virtual-router" and host-route injection based on ARP. This allows all l3leafs to reuse the same subnet.
+              Pointer to interface used for inband management.
+              All configuration must be done using other data models like network services or structured_config.
+              'inband_mgmt_interface' is only used to refer to this interface as source in various management protocol settings (future feature).
+
+              On L2 switches, this defaults to Vlan<inband_mgmt_vlan> if either 'inband_mgmt_subnet' or 'inband_mgmt_ip' is set.
```

Comment:
so when I read this (and only this) - I feel like this is true even when `inband_mgmt_vlan` is not set.

Reply:
`inband_mgmt_vlan` is _always_ set since there is a default value.

Reply:
I know - just the wording makes me think it could not be